Signal/src/views/AvatarImageView.swift

```diff
@@ -1,14 +1,45 @@
-//  Created by Michael Kirk on 12/11/16.
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
 
 import UIKit
 
 @IBDesignable
 class AvatarImageView: UIImageView {
 
-    override func layoutSubviews() {
+    init() {
+        super.init(frame: CGRect.zero)
+        self.configureView()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.configureView()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.configureView()
+    }
+
+    override init(image: UIImage?) {
+        super.init(image: image)
+        self.configureView()
+    }
+
+    func configureView() {
+        self.layer.minificationFilter = kCAFilterTrilinear
+        self.layer.magnificationFilter = kCAFilterTrilinear
+        self.layer.borderWidth = 0.5
         self.layer.masksToBounds = true
-        self.layer.cornerRadius = self.frame.size.width / 2
+        if self.image == nil {
+            self.image = #imageLiteral(resourceName: "empty-group-avatar")
```

Comment:
So, shouldn't we use a different default for non-groups or will this always be overwritten in the 1:1 case?